Make configs more generic

ausmith opened the pull request:
## What does this PR change?
* Makes the templated configs more generic
* Removes company identifying data that is long since dead internally

## Does this PR relate to any other PRs?
* No

## How will this PR impact users?
* It won't

## Does this PR address any GitHub or Zendesk issues?
* No

## How was this PR tested?
* N/A

## Does this PR require changes to documentation?
* No

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Opencost release? If not, why not?
* Not a release impacting change, only modifying the configuration templates

michaelmdresser commented on configs/gcp.json:
```diff
@@ -5,7 +5,7 @@
     "RAM": "0.004237",
     "spotRAM": "0.000892",
-    "projectID": "guestbook-227502",
+    "projectID": "my-awesome-project",
```
Is there a valid string we can provide that has invalid GCP project ID characters to guarantee no overlap with a valid ID? The same question goes for all fields; having something that is too short to be valid also counts, I think!

ausmith replied:
So GCP project IDs must be 6-30 characters in length [source](https://cloud.google.com/resource-manager/docs/creating-managing-projects). AWS S3 bucket names must be between 3 and 63 characters in length [source](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html). And unfortunately AWS Athena table names can be anything less than 255 characters [source](https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html). Values chosen for each accordingly.

michaelmdresser reviewed (commented):
You'll need to set up signoff for the DCO check to pass (requirement from CNCF to have it as a blocker). You can use the provided instructions in the DCO workflow link on this PR. In the longer term, I recommend setting up some sort of automation in your Git-interaction tool of choice to auto-signoff all commits or just commits to your fork of `opencost`.

Commit:
Make configs more generic

Also removing company identifying information and long dead access
keys that should have never been present in these files.

Signed-off-by: Aaron Goldsmith <agoldsmith@kubecost.com>

Commit:
Use invalid values in AWS and GCP object names

Signed-off-by: Aaron Goldsmith <agoldsmith@kubecost.com>

nikovacevic commented:
cc @AjayTripathy can you confirm this won't have any unintended consequences?

michaelmdresser reviewed (approved):
Definitely should have Ajay take a look, but this LGTM. Thanks for addressing feedback as well as possible!

AjayTripathy commented:
its fine, thanks!

ausmith committed:
Merge branch 'develop' into ausmith.remove-dead-access-key
